### ISSUE TITLE
Can't correlate github howto with files in release

### ISSUE THREAD

**Issue by back5576** [OfficeDev/Open-XML-SDK#61]
Github says : "In your program that uses the Open XML SDK, add references to the newly built libraries in build/OpenXmlSdkLib/Debug"
Don't see that. After running the build cmd, I have dll files in:

    Open-XML-SDK\DocumentFormat.OpenXml\bin\release

and

    Open-XML-SDK\DocumentFormat.OpenXml.WB\bin\release

and no explanation as to what "WB" means

**Comment by EricWhiteDev** [comment]
You are right, this is not as clear as it should be.  The WB files is the Open XML SDK that is built using the old System.IO.Packaging that is in the WindowsBase assembly.  We don't recommend using that assembly, because there is the infamous 10MB bug, which is associated with IsolatedStorage.  See the following screen-cast for more information:

http://openxmldeveloper.org/blog/b/openxmldeveloper/archive/2015/06/29/announcing-the-release-of-a-new-system-io-packaging-implementation.aspx

In short, unless you have specific need, don't use that WB files, and use the files in:

Open-XML-SDK\DocumentFormat.OpenXml\bin\release
